sorting/java/InsertionSort.java

```diff
@@ -0,0 +1,29 @@
+

```

Comment:
Rename file's name from `InsertionSort` to `insertion-sort`

Reply:
done